test/test.go

```diff
@@ -28,6 +28,80 @@ func testEmbed(g abstract.Group, rand cipher.Stream, points *[]abstract.Point,
 	*points = append(*points, p)
 }
 
+func testPointSet(g abstract.Group, rand cipher.Stream) {
+	N := 1000
+	null := g.Point().Null()
+	for i := 0; i < N; i++ {
```

Comment:
> Adding randomized tests is in fact good practice

Well, since I'm a rookie I've done a quick search on unit test and I did not find anything about randomized tests. On the contrary, I found reasons to NOT do that: [SO1](https://stackoverflow.com/questions/256065/testing-with-random-inputs-best-practices) and [SO2](https://stackoverflow.com/questions/3441686/what-are-the-downsides-using-random-values-in-unit-testing)
So I still have to disagree with you on that, because I still don't see any good reason to keep the tests like that...